ihatemoney/templates/invitation_mail.fr.j2

```diff
@@ -1,11 +1,11 @@
-Salut,
+Bonjour,
 
-Quelqu'un avec l'adresse "{{ g.project.contact_email }}" vous à invité à partager vos dépenses pour "{{ g.project.name }}".
+Quelqu'un dont l'adresse email est {{ g.project.contact_email }} vous a invité à partager vos dépenses pour « {{ g.project.name }} ».
 
-C'est aussi simple que de dire qui à payé pour quoi, pour qui, et combien celà à coûté, on s’occupe du reste.
+Il suffit de rentrer qui a payé pour quoi, pour qui, et combien ça a coûté, et on s’occupe du reste.
 
-Vous pouvez vous authentifier avec le lien suivant: {{ url_for(".authenticate", _external=True, token=g.project.generate_token()) }}.
-Une fois authentifié, vous pouvez utiliser le lien suivant qui est plus facile à mémoriser: {{ url_for(".list_bills", _external=True) }}
-Si votre cookie est supprimé ou si vous vous déconnectez, vous devrez vous authentifier à nouveau en utilisant le premier lien.
+Vous pouvez vous connecter grâce à ce lien : {{ url_for(".authenticate", _external=True, token=g.project.generate_token()) }}.
+Une fois connecté, vous pourrez utiliser le lien suivant qui est plus facile à mémoriser : {{ url_for(".list_bills", _external=True) }}
+Si votre cookie est supprimé ou si vous vous déconnectez, vous devrez vous connecter à nouveau en utilisant le premier lien.
 
-Have fun,
+En espérant que cet outil vous est utile,
```

Comment:
« vous soit utile » ?

Reply:
- Amusez-vous bien avec IHateMoney
- À bientôt sur IHateMoney
- …

I just think that naming the project is marketing habit. I don't know if we should do it…